src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -42,6 +35,25 @@
      process_metric_update_namespace, process_blob_copy_batch_namespace,
      get_source_file_or_blob_service_client, process_blob_source_uri)
 
+AccountPermissions = get_sdk_attr('azure.multiapi.storage.models#AccountPermissions')
+DeleteSnapshot = get_sdk_attr('azure.multiapi.storage.blob#DeleteSnapshot')
+BlockBlobService = get_sdk_attr('azure.multiapi.storage.blob#BlockBlobService')
+PageBlobService = get_sdk_attr('azure.multiapi.storage.blob#PageBlobService')
+AppendBlobService = get_sdk_attr('azure.multiapi.storage.blob#AppendBlobService')
+BaseBlobService = get_sdk_attr('azure.multiapi.storage.blob.baseblobservice#BaseBlobService')
+BlobContentSettings = get_sdk_attr('azure.multiapi.storage.blob.models#ContentSettings')
+ContainerPermissions = get_sdk_attr('azure.multiapi.storage.blob.models#ContainerPermissions')
+BlobPermissions = get_sdk_attr('azure.multiapi.storage.blob.models#BlobPermissions')
+FileService = get_sdk_attr('azure.multiapi.storage.file#FileService')
+FileContentSettings = get_sdk_attr('azure.multiapi.storage.file.models#ContentSettings')
+SharePermissions = get_sdk_attr('azure.multiapi.storage.file.models#SharePermissions')
+FilePermissions = get_sdk_attr('azure.multiapi.storage.file.models#FilePermissions')
+TableService = get_sdk_attr('azure.multiapi.storage.table#TableService')
+TablePayloadFormat = get_sdk_attr('azure.multiapi.storage.table#TablePayloadFormat')
+QueueService = get_sdk_attr('azure.multiapi.storage.queue#QueueService')
+QueuePermissions = get_sdk_attr('azure.multiapi.storage.queue.models#QueuePermissions')
+PublicAccess = get_sdk_attr('azure.multiapi.storage.blob.models#PublicAccess')
```

Comment:
Why are you not using the "get_versioned_models" to eliminate all these copy-paste redundancy?